client/js/widgets/widget.js

```diff
@@ -984,20 +984,23 @@ export class Widget extends StateManaged {
       }
 
       if(a.func == 'FLIP') {
-        setDefaults(a, { count: 0, face: null, faceCyle: null, collection: 'DEFAULT' });
+        setDefaults(a, { count: 'all', face: null, faceCyle: null, collection: 'DEFAULT' });
+        if(a.count === 'all')
+          a.count = 999999;
+
         let collection;
         if(a.holder !== undefined) {
           if(this.isValidID(a.holder,problems)) {
             await w(a.holder, async holder=>{
-              for(const c of holder.children().slice(0, a.count || 999999))
+              for(const c of holder.children().slice(0, a.count))
```

Comment:
This is sort of unrelated, but if `a.count` is not an integer, something should be pushed to `problems`. Should probably be checked above, before the `if(a.holder...`.
I think this holds for the other functions as well.

Reply:
done